src/utils/utils/schedule.ts

```diff
@@ -72,7 +72,6 @@ export const debounce = <T extends (...args: unknown[]) => void>(
         fn(...latestArgs);
         if (removeScheduler) {
           removeScheduler();
-          removeScheduler = null;
```

Comment:
I admit, I spent a lot of time trying to figure out why sometimes it doesn't work. If you try to install console.log() in hitTest debounce, you will see that it is not deleted from the GlobalScheduler, but is spinning endlessly. Perhaps this is a problem due to StrictMode and the fact that when the Graph is recreated, GlobalScheduler is not recreated. The current fix with a queue for deletion, if anything, does not solve the problem. All in all, there is no need to equate to null, since during iteration there is a check for the presence of an element in the array.